src/components/editor/editor.tsx

```diff
@@ -74,34 +79,34 @@ const PreviewContainer = styled(Box, {
  * @param props component properties
  */
 const Editor = ({ setPanelProperties, surveyId }: Props) => {
-  const [ showAddPage, setShowAddPage ] = useState(false);
+  const [showAddPage, setShowAddPage] = useState(false);
   const setError = useSetAtom(errorAtom);
-  const [ surveyPages, setSurveyPages ] = useAtom(pagesAtom);
-  const [ pageLayouts, setPageLayouts ] = useAtom(layoutsAtom);
-  const [ selectedPage, setSelectedPage ] = useState<number>();
-  const [ isLoading, setIsLoading ] = useState(false);
+  const [surveyPages, setSurveyPages] = useAtom(pagesAtom);
+  const [pageLayouts, setPageLayouts] = useAtom(layoutsAtom);
+  const [selectedPage, setSelectedPage] = useState<number>();
```

Comment:
```suggestion
  const [selectedPageNumber, setSelectedPageNumber] = useState<number>();
```
Would probably make more sense when reading the code.